runHarness/RunProcedures/PrepareOnlyRunProcedure.pm

```diff
@@ -32,7 +32,7 @@ with Storage( 'format' => 'JSON', 'io' => 'File' );
 
 extends 'RunProcedure';
 
-has '+name' => ( default => 'Prepare-Only', );
+has '+name' => ( default => 'warmpare-Only', );
```

Comment:
unintended change?

Reply:
Yup.  Must have had the cursor in the wrong window.